docs/ardana/testing.md

```diff
@@ -210,6 +210,91 @@ prepare_infra
+  to be tested against a cloud input model that includes RHEL (CentOS)
+  compute nodes.
+  * a Gerrit change that touches on functionality that can only be tested
+  on bare-metal deployments (e.g. cobbler) needs to be tested against
+  a QA bare-metal environment in addition, or instead of a virtual one
+
+* the target Gerrit change implements a new piece of functionality for
+which new packages also need to be installed in addition to using a
+different cloud configuration
+
+* the target Gerrit change implements a new piece of functionality for
+which CI coverage is still being developed, but available in a separate
+automation repository fork or branch
+
+* ultimately, the Gerrit change might not require integration testing
+at all, for example because the code changes do not affect any functionality
+areas (e.g. documentation or git configuration changes). For this cases,
+the CI should support skipping the integration testing altogether.
+However, this feature should not be abused.
+
+The Ardana Gerrit CI supports all these use-cases by allowing the users
+to trigger manual Gerrit CI job builds for [cloud 8](https://ci.nue.suse.com/job/openstack-ardana-gerrit-cloud8/build)
+or [cloud 9](https://ci.nue.suse.com/job/openstack-ardana-gerrit-cloud9/build)
+that either complement or replace the default CI builds. The following
+job parameters can be customized to achieve the desired outcome:
+
+* _GERRIT_CHANGE_NUMBER_ - this parameter must be set to the target Gerrit
+change number value
+
+* _gerrit_context_ - this is a string value that uniquely identifies
+a job build within the set of builds running against the same Gerrit
+change. It is included both in the Gerrit build status reports and the
+Jenkins build description. This value also determines which of the
+job builds currently running for a Gerrit change are superseded and
+automatically canceled when a new job build is started. The rule is:
+there can be at most one job with a given _gerrit_context_ value running
+at any given time for a Gerrit change, voting or otherwise.
+
+* _voting_ - this flag controls whether the Gerrit job posts `Verify+2`
+or `Verify-2` label values on the Gerrit change. Non-voting jobs also
+post progress and outcome messages, but do not alter the `Verify` label
+value. There can be at most one voting job running for a Gerrit change
+at any given time. The most recent voting job automatically cancels
+older voting jobs running for the same Gerrit change.
+
+* _integration_test_job_ - the name of an existing Ardana Jenkins job
+that is triggered to run integration tests on the target Gerrit change.
+This parameter may be used to override the default
+`cloud-ardana8-job-std-min-gerrit-x86_64` and `cloud-ardana8-job-std-min-gerrit-x86_64`
+jobs employed by the Ardana CI to run integration tests.
+If the custom integration job also requires its parameters to be overridden,
+this can be achieved via the `extra_params` parameter. If left empty, no
```

Comment:
So this reads as a little confusing to me, but may be more clear in the UI... With the default value be pre-filled in the parameter field in Jenkins? If not and the "default" value in the UI is empty, then that would mean that no job is run?

Reply:
An empty value means no integration testing. The default value is set to point to the std-min job, of course. Updated the docs to make this clearer.